Add missing addr:floor to ALLOWED_LEVEL_TAGS

mnalis opened the pull request:
Fixes https://github.com/Helium314/SCEE/issues/729

Debug `.apk` available at https://github.com/mnalis/StreetComplete/actions/runs/12871833680

- [x] Should be tested if it broke something

mnalis committed:
add  missing addr:floor to ALLOWED_LEVEL_TAGS

mnalis committed:
add debug

mnalis committed:
more debug

mnalis committed:
fix addr:floor typo

mnalis committed:
remove debug

Helium314 commented:
I don't remember the reasoning, but I intentionally left out `addr:floor` from the default allowed level tags.
For fixing the bug, the change to `ADDR_FLOOR("addr:floor)")` is sufficient.

mnalis committed:
do not select addr:floor by default
as indicated in https://github.com/Helium314/SCEE/pull/730#issuecomment-2602768781
